src/VirtoCommerce.PricingModule.Web/Scripts/blades/pricelist-list.js

```diff
@@ -192,7 +192,9 @@ angular.module('virtoCommerce.pricingModule')
             }
 
             filter.change = function () {
-                $localStorage.exportSearchFilterId = filter.current ? filter.current.id : null;
+                $scope.exportSearchFilterId = filter.current ? filter.current.id : null;
```

Comment:
1. Should we save value to scope?
``` js
const exportSearchFilterId = filter.current ? filter.current.id : null;
$localStorage.exportSearchFilterIds[exportDataRequest.exportTypeName] = exportSearchFilterId;
```
2. Should we save the null value instead of deleting it from local storage?

Reply:
1.  Seems this $scope.exportSearchFilterId doesn't make any influence to any cases on the blade. I can't find any. I think it was supposed as clean of $localStorage.exportSearchFilterIds. 
2. Of course we can delete local storage, but where is a profit? This $localStorage.exportSearchFilterIds[exportDataRequest.exportTypeName] will be created while next page reload.